apps/api/src/routes/v1_keys_createKey.ts

```diff
@@ -342,6 +343,14 @@ export const registerV1KeysCreateKey = (app: App) =>
         : Promise.resolve(null),
     ]);
 
+    const defaultPrefixAndBytes = await db.readonly.query.apis.findFirst({
```

Comment:
this is unnecessary to load, we already have the api + keyspace loaded earlier, and also cached :)

https://github.com/unkeyed/unkey/blob/017234bc5f8f75414144ee1b8cc5eb5713084a80/apps/api/src/routes/v1_keys_createKey.ts#L271